### PR TITLE
Support adding more complex text in ControlLabel

### PR DESCRIPTION
**Pull request by badiuoanaalexandra**
Adds support for text to be string or ReactNode

### CODE REVIEW

**Review by miafan23 — APPROVED** [review]
Looks good

File: src/components/control-label/control-label.tsx [hunk 1]
```diff
@@ -33,7 +33,7 @@ ControlLabel.propTypes = {
   /** Value should match the identifying id of the input element. */
   id: PropTypes.string.isRequired,
   /** Label text */
-  text: PropTypes.string.isRequired,
+  text: PropTypes.oneOfType([PropTypes.string, PropTypes.node]),
```
[the remark this answers is not in the document]
Comment (badiuoanaalexandra) [reply src/components/control-label/control-label.tsx:36]: Oh, I didn't see your comment before I merged.. how do I update this..? It is being generated :(

[the remark this answers is not in the document]
Comment (badiuoanaalexandra) [reply src/components/control-label/control-label.tsx:36]: You mean this? 
image

[the remark this answers is not in the document]
Comment (badiuoanaalexandra) [reply src/components/control-label/control-label.tsx:36]: It seems that I cannot change the "union" part, but I added more info in the text itself (I guess that's what you originally meant)
Fix is here https://github.com/mapbox/mr-ui/pull/222

[the remark this answers is not in the document]
Comment (miafan23) [reply src/components/control-label/control-label.tsx:36]: Yes, this what I mean

**Pull request merged**